proxy/src/main/java/com/wavefront/agent/logsharvesting/FilebeatListener.java

```diff
@@ -61,12 +61,10 @@ public FilebeatListener(PointHandler pointHandler, LogsIngestionConfig logsInges
     this.evicted = Metrics.newCounter(new MetricName("logsharvesting", "", "evicted"));
     this.unparsed = Metrics.newCounter(new MetricName("logsharvesting", "", "unparsed"));
     this.parsed = Metrics.newCounter(new MetricName("logsharvesting", "", "parsed"));
-    this.batchesProcessed = Metrics.newCounter(new MetricName("logsharvesting", "", "batchesProcessed"));
 
     // Set up user specified metric harvesting.
     this.metricCache = Caffeine.<MetricName, Metric>newBuilder()
         .softValues()
-        .expireAfterWrite(logsIngestionConfig.aggregationIntervalSeconds, TimeUnit.SECONDS)
```

Comment:
You probably still want an expireAfterAccess to prune old metrics (so that we won't be charging users after a metric stops reporting for, say, an hour)